### PR TITLE
fix(jQuery): weaken window.jQuery error

### LINKED ISSUE #352: Don't throw error if jquery is detected
The error that is thrown here was preventing our app from loading:
https://github.com/TechnologyAdvice/stardust/blob/3d3dab56d1de1a87b946f77c7f04ffc7cfbb48fa/src/utils/jquery.js#L14

jquery might be loaded for any number of reasons. I don't think it's reasonable to throw an error if it exists. Just not using it within stardust should be enough. In our case, we're using stardust in a Meteor app.  Meteor makes it notoriously difficult to remove jquery completely, although we're trying.

### PR DESCRIPTION
**Pull request by dvdzkwsk**
Fixes #352

### CODE REVIEW

File: src/utils/jquery.js [hunk 1]
```diff
@@ -11,15 +11,17 @@ if (window.jQuery) jQuery = window.jQuery
...
     'Do not load jQuery, it is bundled with Stardust.',
     'Stardust will soon be jQuery free: https://github.com/TechnologyAdvice/stardust/issues/247.',
   ].join(' '))
+  /* eslint-enable no-console */
 }
 
 debug('Loading jQuery')
 // load jQuery, then load SUI jQuery
-window.jQuery = window.$ = require('jquery')
+window.jQuery = window.$ = jQuery || require('jquery')
```
Comment (dvdzkwsk) [inline src/utils/jquery.js]: Prevents overriding `window.jQuery` if it exists, which was the source of jQuery extension issues in the past.

**Comment by levithomason** [comment]
Could you confirm the doc site form validation works as expected with this setup?

**Comment by levithomason** [comment]
:accept:

**Comment by dvdzkwsk** [comment]
Confirmed it works with and without jQuery on the window @levithomason .

**Pull request merged**